src/match.js

```diff
@@ -12,14 +12,17 @@ export class Match extends Component {
 	componentWillUnmount() {
 		subscribers.splice(subscribers.indexOf(this.update)>>>0, 1);
 	}
+	isMatch(path) {
+		return path===this.props.path;
+	}
 	render(props) {
 		let url = this.nextUrl || getCurrentUrl(),
 			path = url.replace(/\?.+$/,'');
 		this.nextUrl = null;
 		return props.children[0] && props.children[0]({
 			url,
 			path,
-			matches: path===props.path
+			matches: this.isMatch(path)
```

Comment:
What about allowing `props.isMatch(props, path)` and having the default value be `(props, path) => props.path==path`?

Reply:
Wouldn't it make more sense to reuse `exec` from utils? Chances are you want to use the same pattern for `Match` as you did for `Route`.